using_restricted_array.rb

```diff
@@ -56,6 +62,8 @@ def find_largest(array, length)
   end
   return largest_value
 
+# I wanted to include some of the code I originally tried to create without using the ppt
+# slides. Is this something that would have been ok to use or is the above code a better option?
```

Comment:
This works just fine! At the heart of it, it's doing the same thing only using different language constructs. So, absolutely, go ahead!